.github/workflows/update-wordpress-and-plugins.yml

```diff
@@ -155,10 +155,17 @@ jobs:
           git commit -m $'Update used WooCommerce Memberships plugin in Circle CI\n\n - latest version: ${{ env.WOOCOMMERCE_MEMBERSHIPS_LATEST_VERSION }}\n - previous version: ${{ env.WOOCOMMERCE_MEMBERSHIPS_PREVIOUS_VERSION }}'
 
       # Push all changes at the end if any changes were detected
+      #
+      # For local testing with act tool add following:
+      # env:
+      #   GH_PAT: ${{ secrets.GH_TOKEN }}
+      # run: |
+      #   git remote set-url origin https://${GH_PAT}@github.com/mailpoet/mailpoet
+      #   git push -f origin HEAD:refs/heads/update-plugins-and-wordpress-test
       - name: Push changes
         if: env.CHANGES_DETECTED == 'true'
         run: |
-          git push origin HEAD:refs/heads/update-plugins-and-wordpress
+          git push -f origin HEAD:refs/heads/update-plugins-and-wordpress
 
       # Create a pull request if there are changes
       - name: Create Pull Request
```

Comment:
@costasovo Does this work without an error when a PR already exists? I don't know. Just asking.

Reply:
In my testing with the act tool, it worked. You can see history in the testing PR https://github.com/mailpoet/mailpoet/pull/5761 also the PR desription was updated (I added "UPDATED!") string.

Reply:
Thanks 👍 I'm going to merge it.